docs/components/Flex/Flex.stories.mdx

```diff
@@ -1,12 +1,10 @@
 import { Meta } from "@storybook/addon-docs";
-import { Flex } from "@jobber/components-native";
 
-<Meta title="Components/Layouts and Structure/Flex/Docs" component={Flex} />
```

Comment:
Does removing this do anything? I saw no difference.

Reply:
it used to. not anymore tho because we don't show the props table on mdx